src/external/rawspeed/data/cameras.xml

```diff
@@ -5734,6 +5734,40 @@
 		<Crop x="0" y="0" width="-74" height="0"/>
 		<Sensor black="15" white="3971"/>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-LX15">
+		<ID make="Panasonic" model="DMC-LX15">Panasonic DMC-LX15</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">GREEN</Color>
+			<Color x="1" y="0">BLUE</Color>
+			<Color x="0" y="1">RED</Color>
+			<Color x="1" y="1">GREEN</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="142" white="4095" iso_list="80 100 125 400"/>
```

Comment:
Yes, `dngmeta.rb` is non-deterministic in that regard, https://github.com/darktable-org/darktable/blob/62416056d956a05f891846dc78fc11033d3c699b/tools/dngmeta.rb#L89-L92
If you change `BLACKDIFF_MAX` to `0`, the script then should fail with an error message.
I'm not 100% sure what `dngmeta.rb` should do else if there is more than one black level for a given ISO.